Game/fa/Game.po

```diff
@@ -669,7 +670,7 @@ msgstr "بخش دوم - کامل نشده"
 #: /Game/Resources/Localisation/ST_Menus.ST_Menus
 msgctxt "Menus,GameTitle"
 msgid "ESTRANGED"
-msgstr "ESTRANGED"
+msgstr "دلسرد شده"
```

Comment:
This is the game logo - it should stay as-is in all languages unless there's a good reason to change it